Sources/Tendermint/Types/Transaction.swift

```diff
@@ -6,3 +6,16 @@
 //
 
 import Foundation
+
+
+public typealias TransactionBytes = Data
+
+extension TransactionBytes {
+    var data: Data {
```

Comment:
why add a computed property of data? since you can just use self and have the exact same api no?